src/Blueprint/Blueprint/help/New-AzBlueprint.md

```diff
@@ -0,0 +1,171 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/new-azblueprint
+schema: 2.0.0
+---
+
+# New-AzBlueprint
+
+## SYNOPSIS
+Create a new blueprint and save it within the specified subscription or management group.
+
+## SYNTAX
+
+### CreateBlueprintBySubscription (Default)
+```
+New-AzBlueprint -Name <String> [-SubscriptionId <String>] -BlueprintFile <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### CreateBlueprintByManagementGroup
+```
+New-AzBlueprint -Name <String> -ManagementGroupId <String> -BlueprintFile <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a new blueprint. There are two main ways to create a blueprint: either by providing a blueprint JSON file or providing 
```

Comment:
"Create a new blueprint definition. There are two main ways to create a blueprint definition: either by providing a blueprint definition JSON file or providing"  Use this as the update, but it trails off.  Or providing _what_?